USF-2278: initial draft of changes for removing references to gdrive and sharepoint (#277)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/all/labeling.mdx

### Placeholders file

> A (`placeholders`) content sheet where you change and maintain your storefront's labels, which includes localization. See the [Placeholders file](/resources/placeholders/) for more information.

Reviewer: ```suggestion
A `placeholders` content sheet where you can manage labels for your storefront, which includes localization. See the [Placeholders file](/resources/placeholders/) for more information.
```

### Labeling

> To add or change UI text labels within Commerce drop-in components. Merchants can change the UI labels in their storefronts by changing the values in the `placeholders` file within their content directories.

Reviewer: ```suggestion
Add or change UI text labels within Commerce drop-in components. Merchants can update storefront UI labels by editing the placeholders file in their content directory.
```

### Localizing

> 

Reviewer: Suggest changing the Localizing description to:

Adapt UI text labels and formatting for a specific language. This process includes translating text labels and changing the text direction, date and time formats, and currency symbols to match the target language.
